src/lib/loaders/loaders_with_authentication/impulse.ts

```diff
@@ -48,6 +48,11 @@ export default (accessToken, _userID, opts) => {
       {},
       { method: "POST" }
     ),
+    conversationCreateLoader: impulseLoader(
```

Comment:
I'm surprised we didn't have this already - we have a bunch of conversation-related mutations in MP already. Are all conversations currently only created via inquiry creation workflow? I see [`ConversationService#create_conversation`](https://github.com/artsy/gravity/blob/527f42ce2cd5204f08443ea679e3f9510646d012/app/services/conversation_service.rb#L11) code in Gravity.

Reply:
Yep, we never exposed anything!